mythril/analysis/report.py

```diff
@@ -113,16 +136,45 @@ def as_json(self):
 
     def as_swc_standard_format(self):
         """ Format defined for integration and correlation"""
-        result = {
-            "issues": [
+
+        _issues = []
+        sourceList = []
+
+        for key, issue in self.issues.items():
+
+            if issue.bytecode_hash not in sourceList:
+                idx = len(sourceList)
+                sourceList.append(issue.bytecode_hash)
```

Comment:
If we do this then why bother with the bytecode_hash in the solidity source file?

Reply:
Yes, it might require in solidity source file but not in the issues

Reply:
even if there are no issues i think the bytecode hash needs to be printed in the source list